app/commands/parse_markdown.rb

```diff
@@ -32,18 +32,18 @@ def sanitized_html
 
   memoize
   def raw_html
-    renderer = Renderer.new(options: [:UNSAFE], nofollow_links: nofollow_links)
-    html = CommonMarker.render_doc(
+    doc = CommonMarker.render_doc(
```

Comment:
This is just a naming refactoring. It was previously named `html`, but it is actually containing the parsed document.